Sources/Classes/ViewController.swift

```diff
@@ -58,6 +58,23 @@ public class ViewController: UIViewController {
         backgroundView.backgroundColor = self.backgroundViewColor
         return backgroundView
     }()
+    
+    lazy var captionBackgroundView: UIView = {
+        var captionBackgroundView = UIView()
+        if let captionHeight = captionHeight {
+            var y = view.bounds.height - captionHeight - 32.0
```

Comment:
I don't know what 32.0 is the value for.
Could you change the magic number to the variable?

Reply:
It's the padding of 32 below the caption label: [Line 324](https://github.com/yhkaplan/PhotoSlider/blob/f448f8a55a62c03bac242f7fc8a59ee69256c512/Sources/Classes/ViewController.swift#L324)

I will make it a constant and put a comment above the constant declaration to make it more clear!

Reply:
oops.... ok